### ISSUE TITLE
lbryum help option needs some help

### ISSUE THREAD

**Issue by lbrydocs** [lbryio/lbryum#219]
I just did a fresh install of lbryum and the help command's output is a bit puzzling:

lbryum help

usage: lbryum [-h] [-v] [-P] [-w WALLET_PATH] [-D LBRYUM_PATH] <command> ...
lbryum: error: argument <command>: invalid choice: 'cmd' (choose from 'daemon', 'abandon', 'broadcast', 'cansignwithcertificate', 'check_seed', 'claim', 'claimcertificate', 'claimhistory', 'commands', 'create', 'createmultisig', 'createrawtx', 'decrypt', 'deseed', 'deserialize', 'dumpprivkeys', 'encrypt', 'exportcertificateinfo', 'freeze', 'getaddressbalance', 'getaddresshistory', 'getaddressunspent', 'getalias', 'getbalance', 'getbestblockhash', 'getblock', 'getcertificateclaims', 'getcertificatesforsigning', 'getclaimbyid', 'getclaimbynameinchannel', 'getclaimbyoutpoint', 'getclaimsbyids', 'getclaimsforname', 'getclaimsfromtx', 'getclaimsinchannel', 'getclaimssignedby', 'getclaimtrie', 'getconfig', 'getdefaultcertificate', 'getleastusedaddress', 'getleastusedchangeaddress', 'getmasterprivate', 'getmerkle', 'getmostrecentblocktime', 'getmpk', 'getnameclaims', 'getnetworkstatus', 'getnthclaimforname', 'getprivatekeys', 'getproof', 'getpubkeys', 'getseed', 'getservers', 'getsignaturebyid', 'gettransaction', 'getunusedaddress', 'getutxoaddress', 'getvalueforname', 'getvalueforuri', 'getvaluesforuris', 'history', 'importcertificateinfo', 'importprivkey', 'ismine', 'listaddresses', 'listcontacts', 'listunspent', 'make_seed', 'notify', 'password', 'payto', 'renewclaim', 'renewclaimsbeforeexpiration', 'requestvalueforname', 'restore', 'searchcontacts', 'sendclaimtoaddress', 'sendwithsupport', 'setconfig', 'setlabel', 'signmessage', 'signtransaction', 'support', 'sweep', 'transactionfee', 'unfreeze', 'update', 'updatecertificate', 'updateclaimsignature', 'updatesupport', 'validateaddress', 'verify_claim_schema', 'verifymessage', 'version', 'waitfortxinwallet')

**Comment by kaykurokawa** [comment]
"lbryum --help" instead of "lbryum help" returns a different but a much more informative output . Not sure why there's two different outputs here.

**Comment by eukreign** [comment]
Once lbryum is merged into lbry it'll use the lbry command line --help.
